Pokemon Tabletop Adventure/Supporting Files/JSON/pokédex.json

```diff
@@ -25,9 +25,9 @@
 	            ]
 	        },
 	        "evolution": {
-	            "001": 0,
-	            "002": 15,
-	            "003": 30
+	            "001": "---",
+	            "002": "Minimum level 15",
+	            "003": "Minimum level 30"
```

Comment:
@d9rad 
Suggestions for how to handle the basic evolution stage are welcome - I'm leaning toward either that or something like; `Base Evolution` to indicate there are no requirements for this step.